q2_fragment_insertion/_format.py

```diff
@@ -6,16 +6,86 @@
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
 
+import re
+
+import ijson
+import skbio
+
 import qiime2.plugin.model as model
+from qiime2.plugin import ValidationError
+
+from q2_types.feature_data import AlignedDNAFASTAFormat
+from q2_types.tree import NewickFormat
 
 
 class PlacementsFormat(model.TextFileFormat):
-    def sniff(self):
-        line = open(str(self)).readline()
+    fields = {'tree', 'placements', 'metadata', 'version', 'fields'}
+
+    def _validate_(self, level):
```

Comment:
shall we add a comment about where to find a format description? doi.org/10.1371/journal.pone.0031009

Reply:
Great, thanks for this, will add!